blog-migration.js

```diff
@@ -10,7 +10,8 @@
  * governing permissions and limitations under the License.
  */
 
-import { writeFile, mkdir, access } from 'fs/promises';
+import { writeFile, access } from 'fs/promises';
+import * as fs from 'fs';
```

Comment:
nit: We should probably use `fs.accessSync` and `fs.writeFileSync` instead of mixing them and awaiting.